src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+francecentral;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+japaneast;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+japanwest;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+koreacentral;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+koreasouth;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Sql;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+northcentralus;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+northeurope;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+southafricanorth;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+southcentralus;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+southeastasia;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+southindia;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+uaenorth;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+uksouth;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+ukwest;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+westcentralus;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+westeurope;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+westindia;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+westus;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+westus2;microsoft.sql/locations;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Sql;microsoft.sql/managedinstances;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Sql;microsoft.sql/servers;Azure SQL Database;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.storage/storageaccounts;Storage Accounts;Storage;Microsoft;Cloud Provider;Cloud;IaaS
+MICROSOFT.STORAGE;microsoft.storage/storageaccounts;Storage Accounts;Storage;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.StorageCache;microsoft.storagecache/amlfilesystems;Azure HPC Cache;Storage;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.StorageCache;microsoft.storagecache/caches;Azure HPC Cache;Storage;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.StorageSync;microsoft.storagesync/storagesyncservices;Azure File Sync;Storage;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.StorSimple;microsoft.storsimple/managers;Azure StorSimple;Multicloud;Microsoft;Cloud Provider;Cloud;IaaS

```

Comment:
Should this be Hybrid? I worry we're missing a bunch of Hybrid services 😕

Reply:
All of them hybrid yes